scripts/env/prod.sh

```
@@ -0,0 +1,347 @@
+        log_warning "Gzip compression not detected"
+    fi
+}
+
+# Security headers check
+security_check() {
+    local base_url="${1:-http://localhost:3001}"
+    
+    log_info "Checking security headers for $base_url"
+    
+    # Get headers
+    local headers=$(curl -s -I "$base_url")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)